Token.js

```diff
@@ -268,8 +266,59 @@ class Token {
 
 	}
 
+	/**
+	 * adds a hidden dead cross to tokens
+	 * makes dead cross visible if token has 0 hp
+	 * @param token jquery selected div with the class token
+	 */
+	update_dead_cross(token){
+		if(this.options.max_hp > 0){
+			// add a cross if it doesn't exist
+			if(token.find(".dead").length === 0) 
+				token.prepend(`<div class="dead" hidden></div>`);
+			// update cross scale
+			const deadCross = token.find('.dead')
+			deadCross.attr("style", `transform:scale(${this.get_token_scale()});--size: ${parseInt(this.options.size) / 10}px;`)
+			// check token death
+			if (this.options.max_hp > 0 && parseInt(this.options.hp) === 0) {
+				deadCross.show()
+			} else {
+				deadCross.hide()
+			}
+		}
+	}
+
+	/**
+	 * updates the colour of the health aura if enabled
+	 * @param token jquery selected div with the class token
+	 */
+	update_health_aura(token){
+		console.group("update_health_aura")
+		if ((!this.options.disableaura) && this.options.max_hp > 0) {
+
+			token.find(".token-image").css('box-shadow',
+				`${token_health_aura(
+					Math.round((this.options.hp / this.options.max_hp) * 100)
+				)} 0px 0px 7px 7px`
+			);
+		}
+		console.groupEnd()
+	}
+
+	/**
+	 * returns different scales of the token based on options such as aura disabled
+	 * @returns scale of token
+	 */
+	get_token_scale(){
+		let scale = (((this.options.size - 15) * 100) / this.options.size) / 100;
+			
+		if (!(this.options.max_hp) > 0 || (this.options.disableaura))
```

Comment:
You can clean up some of the parentheses here, too. Looks like they were there before, but as long as you're refactoring, we might as well clean it up a bit.

And as long as you're cleaning things up, I would reverse the logic in this function. Do this check first and return 1 early. That way we don't have to do all that math above if we're not even going to use it.